src/config/config_type.rs

```diff
@@ -66,7 +66,7 @@ impl ConfigType for IgnoreList {
 }
 
 macro_rules! create_config {
-    // Options passed in to the macro.
+    // Options passed into the macro.
```

Comment:
`in to`/`into` has been quite controversial. It's possible that the phrase `pass-in` is being treated as a thing in which case I could understand someone objecting, but, I parse this as **pass** _options_ **into** _container_, and thus `into` is my preference.